quickstart.sh

```diff
@@ -102,6 +102,7 @@ if [ ! "$(docker ps -q -f name=${CONTAINER_NAME})" ]; then
       -e ANSIBLE_COLLECTIONS_PATH="${ANSIBLE_COLLECTIONS_PATH}" \
       -e ANSIBLE_ROLES_PATH="/opt/cldr-runner/roles" \
       -e AWS_DEFAULT_OUTPUT="json" \
+      -v "${PWD}:/opt/cloudera-deploy" \
```

Comment:
What is your specific goal with this change? Presently we do not assume that the quickstart.sh is being run from a full copy of the repo, as it may simply be run independently after wget/curl from github. This is why the parent directory of wherever the quickstart.sh is run from is mounted as /runner/project by default (and may be set during init).

Reply:
That was just an oversight from early experimentation. Removed 👍